docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -319,17 +377,58 @@ variables; worker nodes **must** specify the `CUBESTORE_WORKER_PORT` and
 `CUBESTORE_META_ADDR` environment variables. More information about these
 variables can be found [in the Environment Variables reference][ref-config-env].
 
+A sample Docker Compose stack setting this up might look like:
+
+```yaml
+version: '2.2'
+services:
+  cubestore_router:
+    image: cubejs/cubestore:latest
+    environment:
```

Comment:
We should expose relevant ports: 
```yaml
expose:
  - 9999
  - 3306 # This exposes MySQL endpoint.
  - 3030 # This exposes the HTTP endpoint for CubeJS.
```

Similarly, workers should expose their ports `9001`.

Reply:
@ilya-biryukov @paveltiunov Could you please expand on who is a client for the "MySQL endpoint"? It seems Cube.js communicates with Cube Store via HTTP on 3030 port.

Reply:
@hassankhan let's remove any mention of 3306 port.